build-configuration/android-application.gradle

```diff
@@ -0,0 +1,63 @@
+/**
+ * Base build configuration for all android example applications (modules applying the android application plugin).
+ *
+ * Don't add any configuration here that is not common to *every* android example application.
+ */
+apply plugin: "com.android.application"
+apply plugin: "kotlin-android"
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion rootProject.ext.compileSdkVersion
+        versionCode 11
+        versionName VERSION_NAME
+    }
+
+    signingConfigs {
```

Comment:
This is net new. I've been using this while testing proguard. This makes all developer APKs signed with the same keystore, and signs the release and debug APKs with the same config.

Reply:
This is pretty useful, thanks!